follower/execution/block_data.go

```diff
@@ -21,7 +21,8 @@ import (
 )
 
 type BlockData struct {
-	Block       *flow.Block `validate:"required"`
+	Header      *flow.Header         `validate:"required"`
+	Commit      flow.StateCommitment `validate:"required"`
```

Comment:
We could also check that the length is the expected one here, if we don't want to fully trust the source of block data.

Reply:
I just added and then removed that, since `flow.StateCommitment` is an array now.